de-de/java-de.html.markdown

```diff
@@ -0,0 +1,491 @@
+---
+language: java
+contributors:
+    - ["Michael Dähnert", "http://github.com/JaXt0r"]
+filename: LearnJava-de.java
```

Comment:
please add a `-de` suffix to the filename, and restore the original contributors (add yourself as a translator instead)